lib/msal-common/src/utils/Constants.ts

```diff
@@ -32,6 +32,15 @@ export const Constants = {
     URL_FORM_CONTENT_TYPE: "application/x-www-form-urlencoded"
 };
 
+/**
+ * Denotes which way MSAL is being used per request
+ */
+export enum MsalMethod {
```

Comment:
I think we should not have "Msal" in names of things. This is called "InteractionType" in v1, probably good to agree on "method" vs "interaction type" vs something else.

Reply:
yes. It is something we have been following since the library initial days, no MSAL references within the library nomenclature.

Reply:
sounds good, will change